src/s3fs.cpp

```diff
@@ -970,20 +970,25 @@ static int s3fs_create(const char* _path, mode_t mode, struct fuse_file_info* fi
     }else if(0 != result){
         return result;
     }
-    result = create_file_object(path, mode, pcxt->uid, pcxt->gid);
-    StatCache::getStatCacheData()->DelStat(path);
-    if(result != 0){
-        return result;
+
+    headers_t meta;
+    time_t now = time(NULL);
+    meta["Content-Length"] = "0";
+    meta["x-amz-meta-mtime"] = str(now);
+    meta["x-amz-meta-ctime"] = str(now);
+    meta["x-amz-meta-atime"] = str(now);
+    if(!StatCache::getStatCacheData()->AddStat(path, meta)){
+        return -EIO;
     }
 
     AutoFdEntity autoent;
     FdEntity*    ent;
-    headers_t    meta;
-    get_object_attribute(path, NULL, &meta, true, NULL, true);    // no truncate cache
     if(NULL == (ent = autoent.Open(path, &meta, 0, -1, false, true))){
         StatCache::getStatCacheData()->DelStat(path);
         return -EIO;
     }
+    ent->pagelist.SetPageLoadedStatus(0, 1, PageList::PAGE_LOAD_MODIFIED);
+    ent->is_meta_pending = true;
```

Comment:
This PR is a work-in-progress but I wanted to get your feedback to see if this approach is valid.  I understand that these should modify the fields more cleanly but I did not see such a mechanism?

Reply:
There is no method to directly operate PageList, so if you want to do such an operation, I thinnk it is better to prepare a method of FdEntity.

Reply:
Done.